templates/frontend/pages/editorialMasthead.tpl

```diff
@@ -24,8 +24,8 @@
 						<span class="date_start">{translate key="common.fromUntil" from=$mastheadUser['dateStart'] until=""}</span>
 						<span class="name">
 							{$mastheadUser['user']->getFullName()|escape}
-							{if $mastheadUser['user']->getData('orcid') && $mastheadUser['user']->getData('orcidAccessToken')}
-								<span class="orcid">	
+							{if $mastheadUser['user']->getData('orcid') && $mastheadUser['user']->getData('orcidAccessToken') && $mastheadUser['user']->hasVerifiedOrcid()}
```

Comment:
I think we should remove the check for the `orcidAccessToken` as the addition of `hasVerifiedOrcid` covers this better.